mitodl/open-discussions
# enrollments for single user

#### Pre-Flight checklist

#### What are the relevant tickets?
closes https://github.com/mitodl/open-discussions/issues/3298

#### What's this PR do?
This pr queries for enrollments from the Athena enrollment table for a single user and creates Enrollment records in the open database. It attempts to match the course code from the Athena database (which comes from the mitx bigquery database) to a course run or, failing that, a course.

#### How should this be manually tested?
Ask me for Athena credentials. Set 
ATHENA_ACCESS_KEY_ID
ATHENA_SECRET_ACCESS_KEY
ATHENA_WORK_GROUP
and
ATHENA_REGION_NAME
in your .env file

1) Verify that you can still create a new user through the UI and that you see `Task course_catalog.tasks.update_enrollments_for_email` run on celery after you set the password.

2) Change the email of the user you just created to `pdpinch@mit.edu` Run

```
from course_catalog.tasks import *
update_enrollments_for_email.delay('pdpinch@mit.edu')
```
Verify that 154 enrollments are created
